README.md

```diff
@@ -16,56 +37,16 @@ Twilio Flex plugins allow you to augment/change the behavior and appearance of a
 
 Major changelogs can be found in the [changelog directory](/tree/master/changelog).
 
-## Project Structure
-
-This is a monorepo project managed by [lerna](https://github.com/lerna/lerna) for creating Twilio Flex plugins.
-
-- [create-flex-plugin](packages/create-flex-plugin): The CLI tool to start a new Flex plugin project
-- [flex-plugin](packages/flex-plugin): Runtime dependency for Flex plugins
-- [craco-config-flex-plugin](packages/craco-config-flex-plugin): Config override for [craco](https://github.com/sharegate/craco) used to develop and build Flex plugins
-
-
 ## Contributing
 
-This project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
-
-By cloning this project, you will be contributing to the plugin-builder itself. If you want to create a plugin instead, please instead use the guide in [creating a new plugin](packages/create-flex-plugin/README.md).
-
-### 1. Create a Fork of the Project
+Glad to have you contribute to this project! Please visit [CONTRIBUTING](CONTRIBUTING.md) for more information on how to contribute to this project.
```

Comment:
Check out [CONTRIBUTING](CONTRIBUTING.md) for more information on how to contribute to this project.

Reply:
too many `contribute`s in those sentences